src/control/lib/spdk/nvme.go

```diff
@@ -275,12 +275,18 @@ func processReturn(retPtr *C.struct_ret_t, failMsg string) (ctrlrs []Controller,
 	if retPtr == nil {
 		return nil, errors.Wrap(FaultBindingRetNull, failMsg)
 	}
+
+	defer func() {
+		freeReturn(retPtr)
+	}()
```

Comment:
Don't need to update the PR for this, but I don't think the closure is necessary here. The deferred function won't evaluate retPtr until the defer fires on function exit.

Reply:
good catch